packages/protocol/src/data/task-data.ts

```diff
@@ -35,22 +45,21 @@ export interface TaskDataAccess {
   name: string;
   description: string;
   category: string;
-  'responsible/type': ResponsibleType;
-  'responsible/activator': string;
-  'priority/level': PriorityLevel;
-  'priority/script': string;
+  responsible: Responsible;
+  priority: Priority;
```

Comment:
getting much better: cool.
I think soon expiry should have it's own data object too :D